### DISCUSSION TITLE
Require claim is not working!

### DISCUSSION THREAD

**Discussion by Kacie121** [OpenIDC/mod_auth_openidc#834] [Q&A]
I would like to allow only valid user who also has a role claim with student as a value to access the index page under /test 
The following is what I added to my config file:
**<Location /test>
        AuthType openid-connect
        LogLevel debug
        <RequireAll>
                Require valid-user
                Require claim Role:student
        </RequireAll>
        OIDCUnAutzAction 403
        ErrorDocument 403  "https://<domain>:8811/login/error.html"
</Location>**

When I start the server and hit the endpoint -> https://<domain>:8811/test, it redirects to a login page, then I use a valid account which has student role to login, but after login succeeds, it still redirects to error.html page

However, if I start the server and hit the endpoint -> https://<domain>:8811/login, and after I login successfully, I hit -> https://<domain>:8811/test and it gives me the correct content in /test

I wonder why this would happen. How can I get the content in /test if I hit -> https://<domain>:8811/test directly without going to /login. Please help!

**Comment by zandbelt** [comment]
please provide the version of the module that you're using, the platform that you're on and the server debug logs showing both cases

**Reply by zandbelt** [reply]
since you're using an old and modified version it is impossible to analyze where things go wrong; I'd advise you at least to update your modifications to 2.4.11.1 but wrt. support you're on your own
